js/server.js

```diff
@@ -38,7 +38,7 @@ var Server = function(config, callback) {
 			res.status(403).send("This device is not allowed to access your mirror. <br> Please check your config.js or config.js.sample to change this.");
 		});
 	});
-	app.use(helmet());
+	app.use(helmet(config.helmetOptions));
```

Comment:
@madmod I think we should put the defaults I proposed in. It's not that I disagree with you; the defaults used by helmet include some items that don't apply to us, like HSTS and Key Pinning.